sections/browsers.include

```diff
@@ -2849,49 +2849,49 @@
 
     A static page implementing the x=5 position in such a game could look like the following:
 
-    <pre highlight="html">
-  &lt;!DOCTYPE HTML>
-  &lt;!-- this is https://example.com/line?x=5 -->
-  &lt;title>Line Game - 5&lt;/title>
-  &lt;p>You are at coordinate 5 on the line.&lt;/p>
-  &lt;p>
-  &lt;a href="?x=6">Advance to 6&lt;/a> or
-  &lt;a href="?x=4">retreat to 4&lt;/a>?
-  &lt;/p>
-  </pre>
+    <xmp highlight="html">
+      <!DOCTYPE html>
```

Comment:
The [HTML5 Boilerplate project](https://github.com/h5bp/html5-boilerplate/blob/master/src/index.html#L1) already went through the hot drama of this and ended up going with lowercase conformity (`<!doctype html>`), which has then trickled down into other project. However, if we’re going to kick the dust back up and uppercase part or all of this, I’d hope everyone plays nice and aligns on one defacto.

https://github.com/h5bp/html5-boilerplate/issues/1522

Reply:
My first pass was to make it consistent throughout the examples. Discussion on what it should be is taking place in https://github.com/w3c/html/issues/1193
I'm happy either way.